### PR TITLE
Feature/map button style

### PR DESCRIPTION
**Pull request by Bluesmile82**
- Adapt buttons on the map to the Figma style
- Use dark theme when we change to light basemap
- Update the style and images on the Map settings pop-up

### CODE REVIEW

File: client/src/components/ui/popover.tsx [hunk 1]
```diff
@@ -20,7 +21,7 @@ const PopoverContent = React.forwardRef<
       align={align}
       sideOffset={sideOffset}
       className={cn(
-        'bg-popover text-popover-foreground data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0 data-[state=closed]:zoom-out-95 data-[state=open]:zoom-in-95 data-[side=bottom]:slide-in-from-top-2 data-[side=left]:slide-in-from-right-2 data-[side=right]:slide-in-from-left-2 data-[side=top]:slide-in-from-bottom-2 z-50 w-72 rounded-md border p-4 shadow-md outline-none',
+        'bg-popover text-popover-foreground data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0 data-[state=closed]:zoom-out-95 data-[state=open]:zoom-in-95 data-[side=bottom]:slide-in-from-top-2 data-[side=left]:slide-in-from-right-2 data-[side=right]:slide-in-from-left-2 data-[side=top]:slide-in-from-bottom-2 z-50 w-72 rounded-sm border p-4 outline-none',
```
Comment (clementprdhomme) [inline client/src/components/ui/popover.tsx:24]: The popover shouldn't have rounded corners and no border.

**Comment by Bluesmile82** [comment]
Everything addressed. Thanks for the feedback!

File: client/src/containers/section/map/settings/index.tsx [hunk 2]
```diff
@@ -4,9 +4,10 @@ import Labels from './labels';
 const MapSettings = () => {
   return (
     <div className="space-y-3">
-      <h3 className="font-display text-lg">Map settings</h3>
-
-      <div className="space-y-3 divide-y">
+      <div className="mb-6 flex items-center justify-between">
+        <h3 className="font-serif font-semibold">Map style</h3>
```
[the remark this answers is not in the document]
Comment (Bluesmile82) [reply client/src/containers/section/map/settings/index.tsx]: About the h1 I do agree with this first answer https://stackoverflow.com/questions/38448811/is-it-semantically-correct-to-use-h1-in-a-dialog

**Review by clementprdhomme — COMMENTED** [review]
This is not really part of this task but I'd like to suggest that you also apply this style to the map to avoid showing the demo tiles: https://github.com/Orcasa-Platform/orcasa-review4c/pull/1#discussion_r1319459097

File: client/src/containers/section/map/settings/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Bluesmile82) [reply client/src/containers/section/map/settings/index.tsx]: I wonder if the h1s should be the options on the Navbar. I will add that for now

**Review by clementprdhomme — COMMENTED** [review]
I would avoid defining the navigation elements as titles because then the structure isn't right (see below). Why don't you make “Identify areas of interest for research or interventions.” the only `<h1 />` title and then the dataset groups, `<h2 />`?

The heading tree is incorrect as all the sub-level titles are put under the “Datasets” first level

I'm using an extension called HeadingsMap, in case you want to try it.

By the way, did you see my previous comment about the basemaps? Will you address it in this PR or another one?

Thanks!

**Comment by Bluesmile82** [comment]
> I would avoid defining the navigation elements as titles because then the structure isn't right (see below). Why don't you make “Identify areas of interest for research or interventions.” the only `<h1 />` title and then the dataset groups, `<h2 />`?
> 
> The heading tree is incorrect as all the sub-level titles are put under the “Datasets” first level
> 
> I'm using an extension called HeadingsMap, in case you want to try it.
> 
> By the way, did you see my previous comment about the basemaps? Will you address it in this PR or another one?
> 
> Thanks!

Nice recommendation, thanks. I was hesitant as the h1 is used on the search to define the page and that explanation seemed a bit generic. Ah, of course maybe only the active nav item should have h1. But I don't think is so important for now

**Pull request merged**